src/ZODB/_compat.py

```diff
@@ -12,20 +12,20 @@
 #
 ##############################################################################
 import sys
+from six import PY3
 
 IS_JYTHON = sys.platform.startswith('java')
 
-try:
+
+if not PY3:
```

Comment:
I can use `if PY2` if that seems cleaner. I just used `not PY3` to try to keep the sense of the test the same as the try/except version.